sci-misc/boinc/boinc-7.16.14.ebuild

```diff
@@ -137,8 +137,12 @@ src_install() {
 		# Create new icons. bug 593362
 		local s SIZES=(16 22 24 32 36 48 64 72 96 128 192 256)
 		for s in "${SIZES[@]}"; do
-			convert "${DISTDIR}"/${PN}.tif -resize ${s}x${s} "${WORKDIR}"/boinc_${s}.png || die
-			newicon -s $s "${WORKDIR}"/boinc_${s}.png boinc.png
+			# The convert command is not checked, because it will issue warnings and exit with
+			# an error code if imagemagick is used and was merged with USE="-xml", although the
+			# conversion has worked. See #766093
+			# Instead, check newicon which will fail if the conversion did not produce the icon.
+			convert "${DISTDIR}"/${PN}.tif -resize ${s}x${s} "${WORKDIR}"/boinc_${s}.png
+			newicon -s $s "${WORKDIR}"/boinc_${s}.png boinc.png || die
```

Comment:
`newicon` dies on its own?

Reply:
If the source file does not exists, it completely dies? Then the `|| die` is superfluous.

Reply:
I've taken it out.